### DISCUSSION TITLE
Question on Usage of Pkg with Next.js

### DISCUSSION THREAD

**Discussion by davidnaumann-bastian** [vercel/next.js#49609] [Help]
### Summary

I feel like serving the build file with pkg to exe should be pretty easy but for whatever reason I cannot for the life of me figure out how to do it. Anyone have any advice of compiling nextjs to executables to be deployed natively w/o losing the benefits of next.js along the way?

**Comment by davidnaumann-bastian** [comment]
Works when independently running server.js but getting error when trying to pkg a [custom server](https://nextjs.org/docs/pages/building-your-application/configuring/custom-server) : 


Failed to load next.config.js, see more info here https://nextjs.org/docs/messages/next-config-error
.\node_modules\next\dist\server\config.js:78
                userConfigModule = await import((0, _url).pathToFileURL(path).href);
                                                                             ^

TypeError: Invalid host defined options
    at Object.loadConfig [as default] (.\node_modules\next\dist\server\config.js:78:78)
    at async NextServer.prepare (.\node_modules\next\dist\server\next.js:134:24)

**Reply by AndyFong** [reply]
Hi, I've just met the same problem, have you solve it yet?
